static/app/views/alerts/list/rules/index.tsx

```diff
@@ -61,6 +63,15 @@ class AlertRulesList extends AsyncComponent<Props, State & AsyncComponent['state
     ];
   }
 
+  onRequestSuccess({resp}) {
```

Comment:
@Neo-Zhixing It would be good to explicitly check the `stateKey` value to make sure we don't cause a bug if we use more endpoints here.